app/src/main/java/org/p2p/wallet/utils/NetworkExtensions.kt

```diff
@@ -14,6 +15,13 @@ fun Throwable?.getErrorMessage(resourcesProvider: ResourcesProvider): String =
         else -> this?.message ?: resourcesProvider.getString(R.string.error_general_message)
     }
 
+fun Throwable?.getErrorMessage(context: Context): String =
```

Comment:
we've deleted getErrorMessage with context so we need to use only ResourcesProvider

Reply:
if we do this, we need to inject resourceProvider for base fragment and base activity